database/migrations/2016_05_21_000002_create_customers_table.php

```diff
@@ -16,20 +16,15 @@ public function up() {
             $table->string('phone_nums');
             $table->string('email');
 
-            $table->integer('parent_id')->unsigned()->nullable();
```

Comment:
You should keep this field, just don't keep the foreign line (sorry if I didn't make that clear in my last comment).